### DISCUSSION TITLE
farmers affinity

### DISCUSSION THREAD

**Discussion by gtoilet** [runelite/runelite#15886] [Q&A] [answered]
anyway to get a timer like anitfire/antipoison but for affinity
i go hunting and lose track of what time i got left

**Comment by geheur** [comment, marked as answer]
Fwiw, someone made a PR for this: https://github.com/runelite/runelite/pull/15547

**Comment by gtoilet** [comment]
but im not smart enuf to get his to work and am told unless they implement that into basic install
i would have to make some dev environment ...
